### PR TITLE
feat: Add settings explanations

### PR DESCRIPTION
**Pull request by teolemon**
### What
feat: Add settings explanations

### CODE REVIEW

**Review by stephanegigandet — CHANGES_REQUESTED** [review]
We should not add empty strings for settings explanations, that will have unwanted side effects (e.g. displaying info icons that will lead to a blank screen).

**Comment by teolemon** [comment]
It's still a WIP that I started during a meeting ;-)

File: po/common/common.pot [hunk 1]
```diff
@@ -5234,6 +5246,10 @@ msgctxt "attribute_forest_footprint_setting_name"
 msgid "Low risk of deforestation (Forest footprint)"
 msgstr ""
 
+msgctxt "attribute_forest_footprint_setting_note"
+msgid ""
```
Comment (teolemon) [inline po/common/common.pot:5250]:
```suggestion
msgid "L'empreinte forêt correspond au risque de déforestation associé à ses ingrédients."
```

File: po/common/common.pot [hunk 2]
```diff
@@ -6030,6 +6046,11 @@ msgctxt "maybe_vegan_ingredients"
 msgid "Ingredients that may not be vegan"
 msgstr "Ingredients that may not be vegan"
 
+
+msgctxt "attribute_vegan_setting_note"
+msgid ""
```
Comment (teolemon) [inline po/common/common.pot:6051]:
```suggestion
msgid "To determine whether an ingredient is vegan, we only rely on the list of ingredients."
```

File: po/common/common.pot [hunk 3]
```diff
@@ -6042,6 +6063,10 @@ msgctxt "maybe_vegetarian_ingredients"
 msgid "Ingredients that may not be vegetarian"
 msgstr "Ingredients that may not be vegetarian"
 
+msgctxt "attribute_vegetarian_setting_note"
+msgid ""
```
Comment (teolemon) [inline po/common/common.pot:6067]:
```suggestion
msgid "To determine whether an ingredient is vegetarian, we only rely on the list of ingredients."
```

File: po/common/common.pot [hunk 4]
```diff
@@ -5234,6 +5246,10 @@ msgctxt "attribute_forest_footprint_setting_name"
 msgid "Low risk of deforestation (Forest footprint)"
 msgstr ""
 
+msgctxt "attribute_forest_footprint_setting_note"
+msgid "L'empreinte forêt correspond au risque de déforestation associé à ses ingrédients."
```
Comment (teolemon) [inline po/common/common.pot:5250]:
```suggestion
msgid "The forest footprint corresponds to the risk of deforestation associated with its ingredients."
```

File: po/common/common.pot [hunk 5]
```diff
@@ -6054,6 +6079,10 @@ msgctxt "palm_oil_free_ingredients"
 msgid "No ingredients containing palm oil detected"
 msgstr "No ingredients containing palm oil detected"
 
+msgctxt "attribute_palm_oil_setting_note"
+msgid ""
```
Comment (teolemon) [inline po/common/common.pot:6083]:
```suggestion
msgid "To determine whether an ingredient contains palm oil, we only rely on the list of ingredients."
```

File: po/common/common.pot [hunk 6]
```diff
@@ -6054,6 +6079,10 @@ msgctxt "palm_oil_free_ingredients"
 msgid "No ingredients containing palm oil detected"
 msgstr "No ingredients containing palm oil detected"
 
+msgctxt "attribute_palm_oil_setting_note"
+msgid "To determine whether an ingredient contains palm oil, we only rely on the list of ingredients."
```
Comment (teolemon) [inline po/common/common.pot:6083]:
```suggestion
msgid "To determine whether a product contains palm oil, we only rely on the list of ingredients."
```

File: po/common/common.pot [hunk 7]
```diff
@@ -6042,6 +6063,10 @@ msgctxt "maybe_vegetarian_ingredients"
 msgid "Ingredients that may not be vegetarian"
 msgstr "Ingredients that may not be vegetarian"
 
+msgctxt "attribute_vegetarian_setting_note"
+msgid "To determine whether an ingredient is vegetarian, we only rely on the list of ingredients."
```
Comment (teolemon) [inline po/common/common.pot:6067]:
```suggestion
msgid "To determine whether a product is vegetarian, we only rely on the list of ingredients."
```

File: po/common/common.pot [hunk 8]
```diff
@@ -6030,6 +6046,11 @@ msgctxt "maybe_vegan_ingredients"
 msgid "Ingredients that may not be vegan"
 msgstr "Ingredients that may not be vegan"
 
+
+msgctxt "attribute_vegan_setting_note"
+msgid "To determine whether an ingredient is vegan, we only rely on the list of ingredients."
```
Comment (teolemon) [inline po/common/common.pot:6051]:
```suggestion
msgid "To determine whether a product is vegan, we only rely on the list of ingredients."
```

**Review by stephanegigandet — APPROVED** [review]

File: po/common/common.pot [hunk 9]
```diff
@@ -4683,6 +4683,10 @@ msgctxt "attribute_nova_setting_name"
 msgid "No or little food processing (NOVA group)"
 msgstr "No or little food processing (NOVA group)"
 
+msgctxt "attribute_nova_setting_note"
+msgid "Research suggests a correlation between the consumption of ultra-transformed foods and an increased risk of developing a cancer, among other things."
+msgstr ""
```
Comment (teolemon) [inline po/common/common.pot:4688]:
```suggestion
```

File: po/common/en.po [hunk 10]
```diff
@@ -6067,6 +6079,10 @@ msgctxt "maybe_vegan_ingredients"
 msgid "Ingredients that may not be vegan"
 msgstr "Ingredients that may not be vegan"
 
+msgctxt "attribute_vegan_setting_note"
+msgid "To determine whether a product is vegan, we only rely on the list of ingredients."
+msgstr ""
```
Comment (alexgarel) [inline po/common/en.po:6084]:
```suggestion
msgstr "To determine whether a product is vegan, we only rely on the list of ingredients."
```

File: po/common/en.po [hunk 11]
```diff
@@ -6079,6 +6095,10 @@ msgctxt "maybe_vegetarian_ingredients"
 msgid "Ingredients that may not be vegetarian"
 msgstr "Ingredients that may not be vegetarian"
 
+msgctxt "attribute_vegetarian_setting_note"
+msgid "To determine whether a product is vegetarian, we only rely on the list of ingredients."
+msgstr ""
```
Comment (alexgarel) [inline po/common/en.po:6100]:
```suggestion
msgstr "To determine whether a product is vegetarian, we only rely on the list of ingredients."
```

File: po/common/en.po [hunk 12]
```diff
@@ -6091,6 +6111,10 @@ msgctxt "palm_oil_free_ingredients"
 msgid "No ingredients containing palm oil detected"
 msgstr "No ingredients containing palm oil detected"
 
+msgctxt "attribute_palm_oil_setting_note"
+msgid "To determine whether a product contains palm oil, we only rely on the list of ingredients."
+msgstr ""
```
Comment (alexgarel) [inline po/common/en.po:6116]:
```suggestion
msgstr "To determine whether a product contains palm oil, we only rely on the list of ingredients."
```

**Review by alexgarel — COMMENTED** [review]
fixing

**Pull request merged**